index.js

```diff
@@ -138,13 +138,26 @@ var Node = function (path, opts, disposable) {
     daemonPid: function () {
       return this.subprocess && this.subprocess.pid
     },
-    getConf: function (key, cb) {
+    getConfig: function (key, cb) {
+      if (typeof key === 'function') {
```

Comment:
For the same reason you some times ask a person 'are you sure' and other times 'are you really sure' :D